### PR TITLE
Fix: contrastive logo for the safe mobile wallet

### LINKED ISSUE #546: [ConnectToWeb] Change Safe logo
## Overview
We'd like to change the logo that is received when connecting the web app because it is not visible in dark mode.

## Requirements
Please use the logo from here: https://github.com/safe-global/safe-ios/issues/2606#issuecomment-1242060393

## Links
https://github.com/safe-global/safe-ios/issues/2606

### PR DESCRIPTION
**Pull request by katspaugh**
## What it solves
Resolves #546

## How this PR fixes it
Adds a new logo on a green bg.

## How to test it
Switch the Safe mobile app to dark mode. Connect to the app and check the logo on the device screen.

### CODE REVIEW

File: src/services/pairing/connector.ts [hunk 1]
```diff
@@ -2,18 +2,23 @@ import WalletConnect from '@walletconnect/client'
...
+
   const APP_META = {
     name: `Safe Web v${packageJson.version}`,
-    url: SAFE_REACT_URL,
-    icons: [`${SAFE_REACT_URL}/images/favicons/logo_120x120.png`],
+    url: host,
+    icons: [`${host}${PairingIcon.src}`],
   }
 
+  console.log('APP_META', APP_META)
```
Comment (iamacook) [inline src/services/pairing/connector.ts:20]: Do we want to log this on prod.?

**Review by iamacook — COMMENTED** [review]
I can see the image logged correctly, but it's still not showing for me.

image

File: src/services/pairing/connector.ts (hunk 1 above)
Comment (katspaugh) [reply src/services/pairing/connector.ts]: Removed.

**Comment by katspaugh** [comment]
Yes, same. Investigating.

**Comment by katspaugh** [comment]
I'll go ahead and merge this, as it's not breaking anything. Once the mobile app is fixed, this should work.

**Pull request merged**